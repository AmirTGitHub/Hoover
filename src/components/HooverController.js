import React from 'react';
import arrow from '../assets/arrow.png'

const HooverController = (props) => {
    return (
        <div className='hooverController'>
            <h1>Hoover Controller</h1>
            <button className='up btn-controller' onClick={props.goUp}>
                <img src={arrow} alt='arrow up' />
            </button>
            <div>
                <button className='btn-controller' onClick={props.goLeft}>
                    <img src={arrow} alt='arrow left' />
                </button>
                <button className='right btn-controller' onClick={props.goRight}>
                    <img src={arrow} alt='arrow right' />
                </button>
            </div>

            <button className='down btn-controller' onClick={props.goDown}>
                <img src={arrow} alt='arrow down' />
            </button>

        </div>
    )
}


export default HooverController