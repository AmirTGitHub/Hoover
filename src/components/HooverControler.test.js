import HooverController from './HooverController'


it('render correctly', () => {
    const wrapper = render(<HooverController />)
    expect(wrapper).toMatchSnapshot()
})