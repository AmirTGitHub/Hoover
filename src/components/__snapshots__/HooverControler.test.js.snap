// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  class="hooverController"
>
  <h1>
    Hoover Controller
  </h1>
  <button
    class="up btn-controller"
  >
    <img
      alt="arrow up"
      src="arrow.png"
    >
      
    </img>
  </button>
  <div>
    <button
      class="btn-controller"
    >
      <img
        alt="arrow left"
        src="arrow.png"
      >
        
      </img>
    </button>
    <button
      class="right btn-controller"
    >
      <img
        alt="arrow right"
        src="arrow.png"
      >
        
      </img>
    </button>
  </div>
  <button
    class="down btn-controller"
  >
    <img
      alt="arrow down"
      src="arrow.png"
    >
      
    </img>
  </button>
</div>
`;
